ruby/Gemfile.lock

```diff
@@ -0,0 +1,64 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    addressable (2.8.4)
+      public_suffix (>= 2.0.2, < 6.0)
+    dotenv (2.8.1)
+    faraday (2.7.10)
+      faraday-net_http (>= 2.0, < 3.1)
+      ruby2_keywords (>= 0.0.4)
+    faraday-net_http (3.0.2)
+    licensee (9.15.0)
+      dotenv (~> 2.0)
+      octokit (>= 4.20, < 7.0)
+      reverse_markdown (>= 1, < 3)
+      rugged (>= 0.24, < 2.0)
+      thor (>= 0.19, < 2.0)
+    mustermann (3.0.0)
+      ruby2_keywords (~> 0.0.1)
+    nokogiri (1.15.3-arm64-darwin)
+      racc (~> 1.4)
+    nokogiri (1.15.3-x86_64-linux)
+      racc (~> 1.4)
+    octokit (6.1.1)
+      faraday (>= 1, < 3)
+      sawyer (~> 0.9)
+    public_suffix (5.0.3)
+    racc (1.7.1)
+    rack (2.2.8)
+    rack-protection (3.0.6)
+      rack
+    reverse_markdown (2.1.1)
+      nokogiri
+    ruby2_keywords (0.0.5)
+    rugged (1.6.3)
+    sawyer (0.9.2)
+      addressable (>= 2.3.5)
+      faraday (>= 0.17.3, < 3)
+    sinatra (3.0.6)
+      mustermann (~> 3.0)
+      rack (~> 2.2, >= 2.2.4)
+      rack-protection (= 3.0.6)
+      tilt (~> 2.0)
+    thor (1.2.2)
+    tilt (2.2.0)
+    uri (0.10.3)
+
+PLATFORMS
+  arm64-darwin-22
+  x86_64-linux
+
+DEPENDENCIES
+  licensee
+  sinatra
+  uri (< 0.10.4)
+
+RUBY VERSION
+   ruby 2.7.3p183
+
+
+
+
+
+BUNDLED WITH
+   2.2.1
```

Comment:
:exclamation:**Cycode**: Security vulnerability found in newly introduced dependency. 

||| 
 |---|---|
|**Severity**|High|
|**Issue**|Dependency Confusion in Bundler: CVE-2020-36327|
|**Ecosystem**|RubyGems|
|**Dependency**|`bundler`|
|**Dependency Paths**|``|
|**Direct Dependency**|No|
|**Development Dependency**|No|
|**Upgrade**|`2.2.10`|

 
 Bundler 1.16.0 through 2.2.9 and 2.2.11 through 2.2.17 sometimes chooses a dependency source based on the highest gem version number, which means that a rogue gem found at a public source may be chosen, even if the intended choice was a private gem that is a dependency of another private gem that is explicitly depended on by the application.
 <!-- 0c2685bd-e8ad-4ead-b683-6b8dd70d2737 -->